### PR TITLE
N23: backroll-rs and GGRS

### PR DESCRIPTION
**Pull request by gschup**
_Part of #677_

backroll-rs and GGRS collaborated in a joint section. The section exceeds 1000 characters, but not 2000 characters. Since we are writing for two new libraries, I hope this is okay.

### CODE REVIEW

**Review by 17cupsofcoffee — COMMENTED** [review]
I think going over the character limit is fine here, due to the combo of:

* Multiple projects being covered
* The projects being mentioned for the first time and needing some background info

Only issue I can spot is the line length - the linter (which hasn't run yet, due to GitHub locking down CI for new contributors...) will complain if any line is above 80 characters. Could you add line breaks to resolve this? Happy to approve once that's done.

**Comment by gschup** [comment]
I took care of the line length issue and slightly changed a single sentence to be more precise in regard to the author's recommendations.

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/news/023/index.md [hunk 1]
```diff
@@ -91,6 +91,56 @@ making of [video](https://youtube.com/watch?v=3TOEZ7krhvI) on YouTube.
...
+If you are interested in integrating rollback networking into your game or
+just want to chat with other rollback developers (not limited to Rust),
+check out the [GGPO Developers Discord]!
+
+[backroll-rs]: https://github.com/HouraiTeahouse/backroll-rs
+[bevy-backroll]: https://github.com/HouraiTeahouse/backroll-rs/tree/main/bevy_backroll
+[GGPO Developers Discord]: https://discord.gg/8FKKhCRCCE
+[GGRS]: https://github.com/gschup/ggrs
+[GGPO]: https://www.ggpo.net/
+[@g_schup]: https://twitter.com/
```
Comment (james7132) [inline content/news/023/index.md:141]: This is just a link to the twitter homepage. Pretty sure this should be filled out with an actual profile link.

**Review by james7132 — CHANGES_REQUESTED** [review]

File: content/news/023/index.md (hunk 1 above)
Comment (gschup) [reply content/news/023/index.md:141]: good catch!

**Comment by ozkriff** [comment]
fyi @ValorZard

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**